### PR TITLE
Stratio docs update

### PR DESCRIPTION
**Pull request by iamjanr**
## Description

Update GCP Permissions
Re-write EKS permissions reference - installation.adoc

## Related Pull Requests

N/A

## Pull Request Checklist:

- [ ] [PR title] Include a title referencing a ticket in Jira (e.g. "[CLOUDS-99] Implement a new funcionality").
- [X] [PR desc] Add a summary of the changes made in simple terms.
- [ ] [PR desc] List any pull-request related to this change (docs, tests, feature, etc.).
- [X] [PR labels] Add the corresponding labels (release, skips, cherry-pick, AT-eks-smoke, etc).
- [ ] [Docs] Are changes to the documentation required? (if so, please add references to those PRs).
- [ ] [QA] Are new unit tests required according with the changes? (if so, please add references to those PRs).

### CODE REVIEW

File: stratio-docs/en/modules/ROOT/pages/installation.adoc [hunk 1]
```diff
@@ -8,8 +8,9 @@
 +
 For automated provisioning in AWS/EKS, it is necessary to execute actions on various AWS services such as EC2, ECR, EKS, Elastic Load Balancing (ELB), etc. While the use or lack of use of these actions will depend on the type of installation, the provider validates that the indicated user has these permissions to run normally.
 +
-** xref:attachment$stratio-eks-policy.json[Download permanent permissions for EKS].
-** xref:attachment$stratio-aws-temp-policy.json[Download temporary permissions for EKS and AWS unmanaged].
+** xref:attachment$stratio-aws-unmanaged-policy.json[Descargar permisos permanentes para AWS no gestionado].
+** xref:attachment$stratio-eks-policy.json[Descargar permisos permanentes para EKS].
```
Comment (tperez-stratio) [inline stratio-docs/en/modules/ROOT/pages/installation.adoc:12]:
```suggestion
** xref:attachment$stratio-eks-policy.json[Download permanent permissions for EKS].
```

File: stratio-docs/en/modules/ROOT/pages/installation.adoc [hunk 2]
```diff
@@ -8,8 +8,9 @@
...
+** xref:attachment$stratio-aws-temp-policy.json[Descargar permisos temporales para EKS y AWS no gestionado].
```
Comment (tperez-stratio) [inline stratio-docs/en/modules/ROOT/pages/installation.adoc:13]:
```suggestion
** xref:attachment$stratio-aws-temp-policy.json[Download temporary permissions for EKS and AWS unmanaged].
```

File: stratio-docs/en/modules/ROOT/pages/installation.adoc [hunk 3]
```diff
@@ -8,8 +8,9 @@
 +
 For automated provisioning in AWS/EKS, it is necessary to execute actions on various AWS services such as EC2, ECR, EKS, Elastic Load Balancing (ELB), etc. While the use or lack of use of these actions will depend on the type of installation, the provider validates that the indicated user has these permissions to run normally.
 +
-** xref:attachment$stratio-eks-policy.json[Download permanent permissions for EKS].
-** xref:attachment$stratio-aws-temp-policy.json[Download temporary permissions for EKS and AWS unmanaged].
+** xref:attachment$stratio-aws-unmanaged-policy.json[Descargar permisos permanentes para AWS no gestionado].
```
Comment (tperez-stratio) [inline stratio-docs/en/modules/ROOT/pages/installation.adoc:11]:
```suggestion
** xref:attachment$stratio-aws-unmanaged-policy.json[Download temporary permissions for AWS unmanaged].
```

**Review by tperez-stratio — CHANGES_REQUESTED** [review]
@iamjanr te paso la versión en inglés. Por favor, pide una nueva revisión cuando estén aplicados los cambios. ¡Gracias!

**Review by tperez-stratio — APPROVED** [review]

**Review by dandres-stratio — APPROVED** [review]
lgtm

**Pull request merged**